roles/federation-registry/templates/ams-env.j2

```diff
@@ -1,7 +1,7 @@
-AMS_PROJECT={{federation_registry_ams.project}}
-AMS_BASE_URL={{federation_registry_ams.host}}
-AMS_USER_TOKEN={{federation_registry_ams.token}}
-AMS_ADMIN_TOKEN={{federation_registry_ams.admin_token}}
+AMS_PROJECT={{federation_registry_ams.project | default('')}}
+AMS_BASE_URL={{federation_registry_ams.host | default('')}}
+AMS_USER_TOKEN={{federation_registry_ams.token | default('')}}
+AMS_ADMIN_TOKEN={{federation_registry_ams.admin_token | default('')}}
 EXPRESS_URL=http://localhost:5000
-EXPRESS_KEY={{federation_registry_ams.agent_key}}
+EXPRESS_KEY={{federation_registry_ams.agent_key | default('')}}
 ENV={{federation_registry_deployment_env}}
```

Comment:
If there is no good reason to have empty variables in the env config I'd suggest the following:

```
{% if federation_registry_ams is defined %}
AMS_PROJECT={{ federation_registry_ams.project }}
AMS_BASE_URL={{ federation_registry_ams.host }}
AMS_USER_TOKEN={{ federation_registry_ams.token }}
AMS_ADMIN_TOKEN={{ federation_registry_ams.admin_token }}
{% endif %}
EXPRESS_URL=http://localhost:5000
{% if federation_registry_ams is defined %}
EXPRESS_KEY={{ federation_registry_ams.agent_key }}
{% endif %}
```

Note that the task will fail with an error if `federation_registry_ams` is defined but `federation_registry_ams.project` is not but I guess we want to fail if any of the ams params is missing, right?